db/seeds/jokes.rb

```diff
@@ -33,3 +33,12 @@
 puts Joke.create(lead: "I wouldn't buy anything with velcro.", punchline: "It's all a rip-off!")
 puts Joke.create(lead: "Did you hear about the restaurant on the moon?", punchline: "Great food, but no atmosphere!")
 puts Joke.create(lead: "When you grow up, you'll thank all the sidewalks...", punchline: "for keeping you off the streets!")
+puts Joke.create(lead: "Whats more amazing than a talking dog", punchline: "A Spelling Bee.")
+puts Joke.create(lead: "Why did the snowman call his dog frost", punchline: "Because frost bites.")
+puts Joke.create(lead: "What did the ocean say to the boat?", punchline: "Nothing it just waved.")
+puts Joke.create(lead: "What do you call someone with now body and no nose?", punchline: "Nobody knows!")
+puts Joke.create(lead: "How do the trees feel in spring", punchline: "Releaved!")
+puts Joke.create(lead: "Whats a Plumber's favorite food?", punchline: "Clogs!")
+puts Joke.create(lead: "Did you hear about the fire at the circus", punchline: "It was in tents!")
+puts Joke.create(lead: "Did you hear about the the big paddle sale?", punchline: "It was quite an ordeal!")
```

Comment:
Line is too long. [107/80]<br>Do not write to stdout. Use Rails' logger if you want to log.